usr.sbin/quot/quot.c

```diff
@@ -575,42 +575,47 @@ main(int argc, char *argv[])
 	struct statfs *mp;
 	struct fstab *fs;
 	int cnt;
+	int ch;
+
+	if (argc == 1)
+		usage();
```

Comment:
should this be before or after we parse the args? There's a few 'no sense' combinations that we should likely prohibit and warn about.

like does it make sense to specify -a and also args? 

quot is quite old and also not a lot used, so hasn't had the updating for these sorts of things that other programs have had.

Reply:
Hopefully fixed.